SECloseVoteRequestGenerator.user.js

```diff
@@ -28,7 +28,7 @@
 
 (function(){
     "use strict";
-    
```

Comment:
I think the editor I was using (Atom) did some auto-formatting on me. I wouldn't worry about it though, but I'll revert if it becomes a problem.